en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -93,3 +93,194 @@ To update the properties of a user attribute:
 6. Go to the **Additional Properties** tab and add additional properties that can be used when writing an extension.
 
     ![Edit additional properties]({{base_path}}/assets/img/guides/organization/attributes/edit-attributes-additional-properties.png){: width="500" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+{{ user_attribute_change_verification }}
+
+{% if product_name == "WSO2 Identity Server" %}
+### Try it out
+
+You can try out the follwoing flows using the provided cURLs:
+
+- Update mobile number
+
+    === "Request"
+        ```
+        curl -v -k --user [username]:[password] -X PATCH
+        -d '{
+            "schemas":[],
+            "Operations":[{"op":[operation], 
+            "value":{[attributeName]:[attribute value]}}]
```

Comment:
shall we format this part
better to get the beautified format shown in postman

Reply:
https://github.com/wso2/docs-is/pull/4545/commits/0e6311ec6ed82441ff311dc3555f42d305631f99